src/main/java/raythx98/duke/ui/Ui.java

```diff
@@ -0,0 +1,46 @@
+package raythx98.duke.ui;
+
+/**
+ * Deals with the user interface.
+ */
+public class Ui {
+
+    private String toShowOnScreen = "";
```

Comment:
Perhaps the variable name should be a noun instead?

Reply:
Agreed, will resolve